### PR TITLE
Add php-cs-fixer as dev dependency + cs fix project

### PR DESCRIPTION
**Pull request by kbsali**

### CODE REVIEW

**Comment by dmitry-mukhin** [comment]
uploadcare-php 2017-10-23 14-05-52

**Comment by kbsali** [comment]
screenshot at 2017-10-23 14-07-46

**Comment by kbsali** [comment]
i'm seeing issues with some arrays, reviewing this manually

**Comment by dmitry-mukhin** [comment]
seems that there are some things that still can be easily fixed:
https://travis-ci.org/uploadcare/uploadcare-php/jobs/291514079

**Comment by kbsali** [comment]
@dmitry-mukhin i've fixed more of them, only warnings on line length left...

File: composer.json [hunk 1]
```diff
@@ -36,4 +35,4 @@
         "email": "support@uploadcare.com",
         "issues": "https://github.com/uploadcare/uploadcare-php/issues"
     }
-}
+}
```
Comment (dmitry-mukhin) [inline composer.json]: :(

File: src/Uploadcare/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```
Comment (dmitry-mukhin) [inline src/Uploadcare/.editorconfig]: do you need two of those?

File: composer.json (hunk 1 above)
Comment (kbsali) [reply composer.json]: what's wrong here?

File: src/Uploadcare/.editorconfig (hunk 2 above)
Comment (kbsali) [reply src/Uploadcare/.editorconfig]: nop, mistake

Comment (kbsali) [reply src/Uploadcare/.editorconfig]: removed

File: composer.json (hunk 1 above)
Comment (dmitry-mukhin) [reply composer.json]: no new line at EOF, contradicting your editorconfig

Comment (kbsali) [reply composer.json]: yes, fixed in my last commit

**Comment by dmitry-mukhin** [comment]
Thanks!

**Pull request merged**